indico/modules/events/contributions/fields.py

```diff
@@ -57,16 +57,12 @@ def _serialize_person_link(self, principal, extra_data=None):
 
     def pre_validate(self, form):
         super(ContributionPersonLinkListField, self).pre_validate(form)
-        persons = set()
         for person_link in self.data:
-            if person_link.person in persons:
-                raise ValueError(_("Person with email '{}' is duplicated").format(person_link.person.email))
```

Comment:
from what I understood the change shouldn't allow duplicates to show up anymore and I guess it'll just fail with an integrity error if you tamper with the data to do so anyway. Buf if there's any way to get such duplicates without tampering with data I'd add it back - but then in a parent class; it's not only relevant for contributions